public/app/app.ts

```diff
@@ -166,6 +166,9 @@ function initEchoSrv() {
 
   window.addEventListener('load', (e) => {
     const loadMetricName = 'frontend_boot_load_time_seconds';
+    // should match names in public/views/index-template.html
```

Comment:
nit: I would reword this to say these are 'marked' in public/views/index-template.html to keep with the `performance` API naming.

Reply:
Updated.